packages/mui/src/shared/pickers/MultiLocalizationProvider.test.tsx

```diff
@@ -73,7 +74,11 @@ describe('MultiLocalizationProvider', () => {
       const { result } = renderAllContextHooksWithProps({
         localeOptions: [
           { locale: 'fa', adapter: getLocalizedDateFnsAdapter(Locale.fa) },
-          { locale: 'en', adapter: getLocalizedDateFnsAdapter(Locale.en) },
+          {
+            locale: 'en',
+            adapter: getLocalizedDateFnsAdapter(Locale.en),
+            adapterLocale: enUS,
+          },
```

Comment:
Make an assertion to become sure `adapterLocale` is passed correctly!
You can pass faIR as `adapterLocale` to `en` adapter & check that on `utils` object.